requirements/dev.txt

```diff
@@ -1,11 +1,13 @@
-# SHA1:835718e7a4d50c372a5b00bd3d8713a040e14cb2
-# Please recompile requirements inside the Docker image, not on your local, host machine.
 #
-# To do this, just use the following from your host:
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
 #
-# $ make compile-requirements
+#    $ make compile-requirements
 #
--r prod.txt
```

Comment:
The additions to this file are deps which were previously implied by the import from `prod.in`. As such, we still have some duplicates between `prod.txt` and `dev.txt`

Reply:
Ok so using `-c prod.txt` seems to help with the constraints